iib/workers/config.py

```diff
@@ -36,6 +36,7 @@ class Config(object):
     iib_grpc_init_wait_time: int = 100
     iib_grpc_max_port_tries: int = 100
     iib_grpc_max_tries: int = 5
+    iib_grpc_port_lock_dir = '/tmp/'
```

Comment:
Can we get this by calling `tempfile.gettempdir()` 
Just need to be sure it returns "global" temp and not just the one related for user.

Reply:
Updated to `lock_dir = tempfile.gettempdir()`.